tf-modules/tea-map-cache/main.tf

```diff
@@ -3,6 +3,18 @@ terraform {
     aws = ">= 2.31.0"
   }
 }
+
+resource "aws_vpc_endpoint" "config" {
```

Comment:
What happens here if VPC/lambda subnet IDs is null?     

Looking at the other existing code, it appears that it's (possibly  illegitimately) assuming no VPC subnets === null VPC, so that's not great (and not related to this PR!), but in a situation where VPC is null/no subnets, this would still break.      

This should probably be conditional on vpc_id.

Reply:
Good point!